int16 num
float64 time
float64 forward_velocity
float64 angular_velocity
int16[] subject
float64[] range
float64[] bearing
